### DISCUSSION TITLE
更新日志 - 22.03.5 - 20230804

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#930] [Announcements]
# 系统
1. 支持了广和通（Fibocom）FM650 5G模块

# 应用
* iStore：
  1. 支持隐藏 Docker 插件，对于某些不支持 Docker 的机型用户可能有用
  2. 在使用旧版luci的固件（例如lean的固件）中自动隐藏不支持的luci21插件

# 设备
电犀牛R66s/R68s：固件合并成 r6xs。对于在线升级无影响，只是手动下载固件时需要注意。老固件会继续保留

**Comment by xixiluhui520** [comment]


**Reply by jjm2473** [reply]
看看是不是自己配置了域名绑定。

分别在内网和外网使用nslookup检查域名，看看得到的结果是否一致

**Reply by xixiluhui520** [reply]
明白了，谢谢老大

**Comment by ericsunhl** [comment]
固件升级下载是不是又被限速了，又变得好慢~

**Comment by FlynnYork** [comment]


**Reply by TKKChina** [reply]
我也是

**Reply by FlynnYork** [reply]
有解决方法吗？

**Comment by TKKChina** [comment]
7890是被占用了的




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年8月9日(星期三) 下午2:02
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [istoreos/istoreos] 更新日志 - 22.03.5 - 20230804 (Discussion #930)





 
OpenClash启动正常，但是始终连接不上本机IP地址，7890连接不了，老大教教我应该怎样做！
 
 
 
&mdash;

**Comment by TKKChina** [comment]
相当代理了，你可以设置自己端口



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年8月10日(星期四) 上午9:46
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [istoreos/istoreos] 更新日志 - 22.03.5 - 20230804 (Discussion #930)





 
那应该也不会那么多端口都被占吧！！

**Comment by TKKChina** [comment]
你是在软路由里面搞得吗？浏览器访问外网可以吗？



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年8月10日(星期四) 上午9:59
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [istoreos/istoreos] 更新日志 - 22.03.5 - 20230804 (Discussion #930)





 
听你的换了端口，系统端口7898也给了它，但是结果是一模一样，7898压根连接不上，应该是clash的内核不知道为什么不兼容了，passwall能够正常使用！

**Comment by TKKChina** [comment]
我这规则的上网的我用外网是正常的





------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年8月10日(星期四) 上午10:06
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [istoreos/istoreos] 更新日志 - 22.03.5 - 20230804 (Discussion #930)





 
当然啊istoreOS安装的呀！clash开启后在家里是可以访问油管什么的，但是网速是完全不正常的，Nekogram(GT群）直连是连接不上的，开启代理端口一样连接不上。
 如果开启passwall来代理就突然全部正常了！

**Comment by TKKChina** [comment]
我没用passwall，国外网站我一样可以很正常，Openclash也是正常运行



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年8月10日(星期四) 上午10:16
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [istoreos/istoreos] 更新日志 - 22.03.5 - 20230804 (Discussion #930)





 
不知道你说的外网正常是指的什么！反正我国外网站用passwall很正常。用Openclash就不正常

**Comment by TKKChina** [comment]
这里搞好没






------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年8月10日(星期四) 上午10:23
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [istoreos/istoreos] 更新日志 - 22.03.5 - 20230804 (Discussion #930)





 
不过passwall在自行添加某些域名走代理或者不走代理方便要比Openclash方便很多！

**Comment by soundlift** [comment]
UPnP 好像用不了了，大家的能用吗

**Reply by xixiluhui520** [reply]
正常

**Reply by jjm2473** [reply]
在upnp的高级设置里面开启强制转发，并且关掉里面的stun试试

**Reply by soundlift** [reply]
嗯  是stun的问题，关闭就好了

**Comment by xixiluhui520** [comment]
openclash问题已经解决，是etc/config/openclash这个文件问题，把它删除，再删除openclash的ipk，在重新安装openclash与配置就正常了！

**Reply by jjm2473** [reply]
那应该就是新老版本的配置不兼容，要么就是之前的配置文件损坏了

**Reply by xixiluhui520** [reply]
应该是新老版本不兼容，因为一直没有怎么动哪些配置，如果跟我一样也碰到了这样情况的朋友们可以 跟我一样处理即可解决。
手动删除etc/config/openclash这个文件，删除etc/openclash文件夹，再去系统-软件包 卸载openclash的ipk。最好重新安装openclash重新配置即可完全正常食用openclash了！
以上说的是openclash突然不能连接7890端口，openclash运行正常但是实际无法代理的BUG！

**Comment by leige87** [comment]
不知道为什么用这个系统大概1个月后就会出现延迟丢包的情况，我测试过好几次都这样，搞得我不太敢用了，一个月左右网络准卡顿

**Reply by xixiluhui520** [reply]
应该不是系统问题

**Reply by jjm2473** [reply]
看看内存和CPU占用情况

**Comment by FSDXC111** [comment]
R2c报错但正常使用!可否修改一下[6bad8926-4a36-4ca4-80a8-a97de41714db.png](https://github.com/istoreos/istoreos/assets/128386077/2c458811-8057-4186-b5e0-5517e6cf08ae)

**Reply by jjm2473** [reply]
试试chrome浏览器

**Reply by FSDXC111** [reply]
> 试试chrome浏览器

都用了，都是一样的

**Reply by jjm2473** [reply]
我用r2s测试了，没有任何问题，还特意安装nginx测试了。r2s跟r2c是同一个固件。你遇到的问题肯定是因为使用了代理或者内网穿透之类的东西，又没透传404响应

**Reply by FSDXC111** [reply]
> 响应

但是直接重装打开接口那里也是这样，啥都没开

**Reply by jjm2473** [reply]
我不是问你改了什么，我是问你从哪里访问的，是不是内网，是不是有其他的代理。最简单的，恢复出厂设置，电脑连lan口，然后用chrome浏览器打开 http://192.168.100.1/ ，如果能出现同样的错误，再来反馈

**Reply by FSDXC111** [reply]
> 我不是问你改了什么，我是问你从哪里访问的，是不是内网，是不是有其他的代理。最简单的，恢复出厂设置，电脑连lan口，然后用chrome浏览器打开 http://192.168.100.1/ ，如果能出现同样的错误，再来反馈

所有浏览器都这样

**Reply by FSDXC111** [reply]
> 我不是问你改了什么，我是问你从哪里访问的，是不是内网，是不是有其他的代理。最简单的，恢复出厂设置，电脑连lan口，然后用chrome浏览器打开 http://192.168.100.1/ ，如果能出现同样的错误，再来反馈

而且最开始我都说了所有浏览器都一样

**Reply by jjm2473** [reply]
是内网访问的吗

**Comment by xxz296** [comment]
R2S 刷完后，内网不定时停止工作，LAN灯熄灭。只能重启，请教可能是什么原因？

**Reply by jjm2473** [reply]
配置下日志持久化，重启以后才能看到之前的日志。如果有串口调试那就更方便了。

**Reply by jjm2473** [reply]
检查下系统日志和内核日志，使用过程中有没有什么异常。还有恢复出厂设置或者重做系统看看能不能解决
